scripts/core_tests_list.json

```diff
@@ -13,6 +13,7 @@
     "test_icp",
     "test_ifsck",
     "test_ils",
+    "test_error_handling",
```

Comment:
Should this be `test_imeta_error_handling`?

Reply:
looks like it